src/web/src/components/Opportunity/OpportunityPublicSmall.tsx

```diff
@@ -29,10 +29,10 @@ const OpportunityPublicSmallComponent: React.FC<InputProps> = ({
         <div className="flex flex-row">
           <div className="flex flex-row">
             <div className="flex w-3/5 flex-grow flex-col">
-              <h1 className="h-[32px] max-w-[180px] overflow-hidden text-ellipsis text-xs font-medium text-gray-dark md:max-w-[200px]">
+              <h1 className="h-[32px] max-w-[180px] overflow-hidden text-ellipsis text-xs font-medium text-gray-dark md:max-w-[160px] xl:max-w-[200px]">
```

Comment:
it may be better to use the "truncate" class instead of using fixed widths to stop text overflowing. i have noticed that the "truncate" class doesn't work it you are using grids... it only works with flex. 

for example:
  <div className="flex flex-grow flex-col pl-2">
              <div className="truncate text-sm font-semibold">
                {selectedRow?.name}
              </div>
              <div className="truncate text-xs">{selectedRow?.description}</div>
            </div>